.evergreen/buildvariants-and-tasks.in.yml

```diff
@@ -60,6 +60,40 @@ const PACKAGE_BUILD_VARIANTS = [
   }
 ];
 
+const SMOKETEST_BUILD_VARIANTS = [
+  {
+    name: 'smoketest-ubuntu',
+    display_name: 'Smoketest Ubuntu',
+    run_on: 'ubuntu2004-large',
+    depends_on: 'package-ubuntu',
+  },
+  
+//  {
```

Comment:
For now I have most platforms commented out because this is a lot of new tasks that will do nothing for the moment.